src/containers/FormProfile/FormProfile.tsx

```diff
@@ -19,29 +17,22 @@ const FIELDS = {
   [UserUpdateRequestKeys.email]: formFieldsDictionary.email,
 };
 
-export function FormProfile({ userData }: FormProfileProps) {
+export function FormProfile() {
+  const { data } = profileApi.useGetProfileQuery();
+  const [updateProfile, { isLoading }] = profileApi.useUpdateProfileMutation();
   const navigate = useNavigate();
   const fields = useMemo<FormFieldProps[]>(() => {
```

Comment:
🧐
Разные по смыслу хуки лучше отделять, что всё в кашу не лепилось
```suggestion
  const [updateProfile, { isLoading }] = profileApi.useUpdateProfileMutation();
  
  const navigate = useNavigate();
  
  const fields = useMemo<FormFieldProps[]>(() => {
```

Reply:
В компоненте выше тот же поинт, там гитхаб не даёт адекватный коммент сделать

Reply:
Блин, не хочется в ручную это все помнить и делать, если получится линтер настроить поправлю везде сразу

Reply:
Если сконфижишь линтер будет огонь, но это в целом полезно иметь в виду, когда пишешь компонент